api/batch.go

```diff
@@ -109,7 +109,8 @@ func (a *API) getFullBatch(c *gin.Context) {
 	// Fetch txs forged in the batch from historyDB
 	maxTxsPerBatch := uint(2048) //nolint:gomnd
 	txs, _, err := a.h.GetTxsAPI(
-		nil, nil, nil, nil, batchNum, nil, nil, nil, &maxTxsPerBatch, historydb.OrderAsc,
+		nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, batchNum,
```

Comment:
I think it would be good to wait until this PR is approved https://github.com/hermeznetwork/hermez-node/pull/699 to update the `GetTxsAPI` to use a struct parameter.